io.catenax.single_level_bom_as_specified/2.0.1/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,167 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.single_level_bom_as_specified:2.0.1#>.
+
+:SingleLevelBomAsSpecified a samm:Aspect;
+    samm:preferredName "Single Level BOM as Specified"@en;
+    samm:description "The SingleLevelBomAsSpecified defines the view of the OEM or producer of the whole product, e.g. the OEM of a vehicle. It is free of any supplier-related information and specifies the promised and guaranteed content of the whole product to the end customer. This \"top-down\" view is in contrast to the \"bottom-up\" view of the SingleLevelBoMAsPlanned, though several sub-aspects are shared. The BomAsSpecified is merely one aspect, which is attached to the twin of the whole product and itself does neither introduce further twins nor reference them. Instead it merely comprises all functional information required by dismantlers, workshops or requestors for used parts to search for and to make a match on the market place."@en;
+    samm:properties (:assetId :childItems :manufacturerId);
+    samm:operations ();
+    samm:events ().
+:assetId a samm:Property;
+    samm:preferredName "assetID"@en;
+    samm:description "A unique reference to an asset which could be registered within Catena-X DataSpace"@en;
+    samm:characteristic :assetIdTrait;
+    samm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d".
+:manufacturerId a samm:Property;
+    samm:preferredName "Manufacturer ID"@en;
+    samm:description "The ID of the initial issuer of the single level BoMAsSpecified"@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "BPNL1234567890LR".
+:assetIdTrait a samm-c:Trait;
```

Comment:
for uuidv4, you can directly make use of shared aspect model. example: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR45
and then you can delete your own Characteristic and Uuidv4RegularExpression